libqtile/core/manager.py

```diff
@@ -550,15 +550,10 @@ def free_reserved_space(
         # mypy can't work out that the new tuple is also length 4 (see mypy #7509)
         self.reserve_space(tuple(-i for i in reserved_space), screen)  # type: ignore
 
-    def map_window(self, win: base.WindowType) -> None:
-        self.manage(win)
-        if not win.group or not win.group.screen:
-            return
-        win.unhide()
```

Comment:
i should clarify this has been (re)moved because it was only used in one place, by Core.handle_MapNotify, so as part of the mypy-related changes the logic has just moved to that method.